openedx/core/lib/edx_api_utils.py

```diff
@@ -0,0 +1,68 @@
+"""Helper functions to get data from APIs"""
+from __future__ import unicode_literals
+import logging
+
+from django.core.cache import cache
+from edx_rest_api_client.client import EdxRestApiClient
+
+from openedx.core.lib.token_utils import get_id_token
+
+
+log = logging.getLogger(__name__)
+
+
+def get_edx_api_data(api_config, user, resource, querystring=None, cache_key=None):
+    """Fetch the data from the API using provided API Configuration and
+    resource.
+
+    Arguments:
+        api_config(Api_configuration): The configuration model governing
```

Comment:
Minor: put a space between the argument name and the type. Also, type here should be ConfigurationModel.